keps/sig-node/3960-pod-lifecycle-sleep-action/README.md

```diff
@@ -47,10 +47,10 @@ Items marked with (R) are required *prior to targeting to a milestone / release*
 - [x] (R) Design details are appropriately documented
 - [x] (R) Test plan is in place, giving consideration to SIG Architecture and SIG Testing input (including test refactors)
   - [x] e2e Tests for all Beta API Operations (endpoints)
```

Comment:
tests mentioned later in the doc are still pointing to the alpha job: https://testgrid.k8s.io/presubmits-kubernetes-nonblocking#pull-kubernetes-e2e-gce-cos-alpha-features Why alpha? Is there a better link?

Reply:
So although the test name contains `alpha`, it actually runs many non-alpha feature test.
We mentioned this test in the doc because we added this specific fg to the test config(https://github.com/kubernetes/test-infra/pull/31959), so that the test for this feature can be run periodically. It doesn't care if the feature is in alpha stage or beta stage.
We can change the link, but this should be done after this fature is GAed, and the tests are run periodically in another job?  
For now, I think this is the only job which periodically run the test for this feature.